shesha-reactjs/src/components/dataTable/index.tsx

```diff
@@ -318,6 +335,11 @@ export const DataTable: FC<Partial<IIndexTableProps>> = ({
             : undefined;
         const width = strictWidth ?? columnItem.width;
 
+        const isFixed =
+          (['action', 'crud-operations'].includes(columnItem.columnType) && hasFixedColumns) || columnItem?.isFixed;
```

Comment:
@nthangeniphumudzo action and crud-operations columns must work according to their settings without any non-trivial logic. It's not transparent why these columns are fixed when you have any other fixed columns